<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1178</width>
    <height>576</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Gestion des Clients</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>1141</width>
      <height>511</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image:linear-gradient(to right,#141e30  ,#243b55) ;
            </string>
    </property>
    <property name="currentIndex">
     <number>4</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>ajout</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>371</width>
        <height>431</height>
       </rect>
      </property>
      <property name="title">
       <string>Ajouter Colis</string>
      </property>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>40</y>
         <width>71</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Id</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="le">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>40</y>
         <width>121</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="B_ajouter">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>270</y>
         <width>131</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string>Ajouter</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>90</y>
         <width>71</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>poids</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="poids">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>80</y>
         <width>121</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="volume">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>120</y>
         <width>121</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>120</y>
         <width>71</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>volume</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="etat">
       <property name="geometry">
        <rect>
         <x>230</x>
         <y>170</y>
         <width>121</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>170</y>
         <width>161</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>etat: casse,non casse</string>
       </property>
      </widget>
      <widget class="QPushButton" name="mod1">
       <property name="geometry">
        <rect>
         <x>200</x>
         <y>270</y>
         <width>131</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string>modifier</string>
       </property>
      </widget>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>10</y>
        <width>201</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string>LISTE DES COLIS</string>
      </property>
     </widget>
     <widget class="QTableView" name="tab_ajout">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>50</y>
        <width>711</width>
        <height>341</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="Rech_3">
      <property name="geometry">
       <rect>
        <x>860</x>
        <y>20</y>
        <width>201</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="Rech">
      <property name="geometry">
       <rect>
        <x>730</x>
        <y>20</y>
        <width>131</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Rechercher</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pdf">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>400</y>
        <width>91</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="DE">
      <property name="geometry">
       <rect>
        <x>1040</x>
        <y>90</y>
        <width>51</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab2">
     <attribute name="title">
      <string>Elimineruncolis</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>30</y>
        <width>491</width>
        <height>331</height>
       </rect>
      </property>
      <property name="title">
       <string>Ajouter Colis</string>
      </property>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>50</y>
         <width>71</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Id</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="supp_2">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>50</y>
         <width>121</width>
         <height>31</height>
        </rect>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">border-radias:8px

</string>
       </property>
      </widget>
      <widget class="QPushButton" name="supp">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>130</y>
         <width>131</width>
         <height>41</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">height: 50px;
                width:280px;
                font-size: 20px;
                font-family: Araial;
                font-weight: bold;
                background-color: lightgrey;
                border: none;
                border-radius: 8px;</string>
       </property>
       <property name="text">
        <string>Supprimer</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>affichage</string>
     </attribute>
     <widget class="QTableView" name="tab_c">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>671</width>
        <height>341</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="aff">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>360</y>
        <width>131</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Afficher</string>
      </property>
     </widget>
     <widget class="QComboBox" name="combo">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>410</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>poids</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>volume</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>id</string>
       </property>
      </item>
     </widget>
     <widget class="QPushButton" name="tri_b">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>410</y>
        <width>81</width>
        <height>34</height>
       </rect>
      </property>
      <property name="text">
       <string>tri</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_4">
     <attribute name="title">
      <string>Map</string>
     </attribute>
     <widget class="QQuickWidget" name="quickWidget_MapView">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>611</width>
        <height>451</height>
       </rect>
      </property>
      <property name="resizeMode">
       <enum>QQuickWidget::SizeRootObjectToView</enum>
      </property>
     </widget>
     <widget class="QPushButton" name="Rech_2">
      <property name="geometry">
       <rect>
        <x>770</x>
        <y>270</y>
        <width>131</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Rechercher</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="Rech_l_2">
      <property name="geometry">
       <rect>
        <x>730</x>
        <y>220</y>
        <width>201</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>750</x>
        <y>110</y>
        <width>251</width>
        <height>101</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>suivi de colis</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_5">
     <attribute name="title">
      <string>stats</string>
     </attribute>
     <widget class="QPushButton" name="stat">
      <property name="geometry">
       <rect>
        <x>480</x>
        <y>390</y>
        <width>131</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>stat</string>
      </property>
     </widget>
     <widget class="QLabel" name="num1">
      <property name="geometry">
       <rect>
        <x>1010</x>
        <y>430</y>
        <width>111</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QWidget" name="verticalLayoutWidget_3">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>28</y>
        <width>691</width>
        <height>331</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3"/>
     </widget>
     <widget class="QWidget" name="verticalLayoutWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>357</width>
        <height>317</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout"/>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QQuickWidget</class>
   <extends>QWidget</extends>
   <header location="global">QtQuickWidgets/QQuickWidget</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
